src/styles/button.css

```
@@ -0,0 +1,42 @@
+.information__link {
+  margin-top: 16px;
+
+  display: flex;
+  text-decoration: none;
+}
+
+.information__link_button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+
+  padding-top: 2px;
+  height: 40px;
+
+  background: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  letter-spacing: 0;
+
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.information__link_button:hover {
+  background: #fff;
+  border-color: #00acdc;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #00acdc;
+
+  text-align: center;
+  text-transform: uppercase;
```

you already have this styles set in previous class, no need to set it for hovered state
```suggestion
  color: #00acdc;
```